html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form id = 'coding-form'>
+     <div>Email:</div>
+     <label for="email">
+      <input type="email" id="email" name="user_email">
+     </label>
```

Comment:
`label` elements need to have some text associated with it. Your text inside your `div` on line 10, should go inside this `label`